src/core/src/CoreMain.py

```diff
@@ -131,6 +131,7 @@ def __init__(self, argv):
             composite_logger.log_debug("Completed exception handling.\n")
 
         finally:
+            status_handler.log_truncated_patches()
```

Comment:
status_handler is not guaranteed to be initialized in this finally block. 

See line 118 on the pattern to use.